include/seqan3/alignment/all.hpp

```diff
@@ -78,18 +78,18 @@
  * types cannot be printed within the static assert, but the following table shows which combinations are possible.
  * In general, the same configuration element cannot occur more than once inside of a configuration specification.
  *
- * | **Config**                                              | **0** | **1** | **2** | **3** | **4** | **5** | **6** | **7** | **8** | **9** |
- * | --------------------------------------------------------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
- * | \ref seqan3::align_cfg::aligned_ends "0: Aligned ends"  |   ❌   |   ✅   |  ✅    |  ❌    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::align_cfg::band_fixed_size "1: Band"       |       |   ❌   |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::align_cfg::gap "2: Gap scheme"             |       |       |   ❌   |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::global_alignment "3: Global alignment"     |       |       |       |  ❌    |  ❌    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::local_alignment "4: Local alignment"       |       |       |       |       |  ❌    |  ❌    |  ✅    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::align_cfg::max_error "5: Max error"        |       |       |       |       |       |  ❌    |  ✅    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::align_cfg::parallel "6: Parallel"          |       |       |       |       |       |       |  ❌    |  ✅    |  ✅    |  ✅    |
- * | \ref seqan3::align_cfg::result "7: Result"              |       |       |       |       |       |       |       |  ❌    |  ✅    |  ✅    |
- * | \ref seqan3::align_cfg::scoring "8: Scoring scheme"     |       |       |       |       |       |       |       |        |   ❌   |  ✅   |
- * | \ref seqan3::align_cfg::vectorise "9: Vectorise"        |       |       |       |       |       |       |       |        |        |  ❌   |
+ * | **Config**                                               | **0** | **1** | **2** | **3** | **4** | **5** | **6** | **7** | **8** | **9** |
+ * | ---------------------------------------------------------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
+ * | \ref seqan3::align_cfg::aligned_ends "0: Aligned ends"   |   ❌   |   ✅   |  ✅    |  ✅    |  ✅    |  ❌    |  ✅    |  ✅    |  ✅    |  ✅     |
+ * | \ref seqan3::align_cfg::band "1: Band"                   |       |   ❌   |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅     |
+ * | \ref seqan3::align_cfg::gap "2: Gap scheme"              |       |       |   ❌   |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅    |  ✅     |
+ * | \ref seqan3::align_cfg::max_error "3: Max error"         |       |       |       |  ❌    |  ✅    |  ❌    |  ✅    |  ✅    |  ✅    |  ✅     |
```

Comment:
Why did the checks for the max error change?

Reply:
In include/seqan3/alignment/configuration/detail.hpp, `seqan:3:align_cfg::max_error` was already incompatible with the local alignment mode, but the incompatibility was not stated here. That is fixed now.